Tests/SnapshotConfiguration.swift

```diff
@@ -47,7 +47,6 @@ extension SnapshotConfiguration {
     "Nonanimating/FirstText": .precision(0.99),
     "Nonanimating/verifyLineHeight": .precision(0.99),
     "Nonanimating/blend_mode_test": .precision(0.99),
-    "Issues/issue_885": .precision(0.99),
```

Comment:
I removed this animation because its been flaky and showing up in diffs every time I regenerate snapshots with `isRecording = true`